patches/sowm-primitive-multimonitor.patch

```diff
@@ -0,0 +1,94 @@
+diff --git a/Makefile b/Makefile
+index 8573837..72e9542 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,5 +1,5 @@
+ CFLAGS+= -std=c99 -Wall -Wextra -pedantic
+-LDADD+= -lX11
++LDADD+= -lX11 -lXinerama
+ LDFLAGS=
+ PREFIX?= /usr
+ BINDIR?= $(PREFIX)/bin
+diff --git a/sowm.c b/sowm.c
+index 0cc1293..40b39dd 100644
+--- a/sowm.c
++++ b/sowm.c
+@@ -4,6 +4,7 @@
+ #include <X11/XF86keysym.h>
+ #include <X11/keysym.h>
+ #include <X11/XKBlib.h>
++#include <X11/extensions/Xinerama.h>
+ #include <stdlib.h>
+ #include <signal.h>
+ #include <unistd.h>
+@@ -48,7 +49,7 @@ static void ws_go(const Arg arg);
+ static int  xerror() { return 0;}
+ 
+ static client       *list = {0}, *ws_list[10] = {0}, *cur;
+-static int          ws = 1, sw, sh, wx, wy;
++static int          ws = 1, sw, sh, wx, wy, monitors;
+ static unsigned int ww, wh;
+ 
+ static Display      *d;
+@@ -108,6 +109,7 @@ void notify_motion(XEvent *e) {
+         wy + (mouse.button == 1 ? yd : 0),
+         ww + (mouse.button == 3 ? xd : 0),
+         wh + (mouse.button == 3 ? yd : 0));
++    win_size(cur->w, &cur->wx, &cur->wy, &cur->ww, &cur->wh);
+ }
+ 
+ void key_press(XEvent *e) {
+@@ -172,12 +174,41 @@ void win_kill() {
+     if (cur) XKillClient(d, cur->w);
+ }
+ 
++int coords_in_box (int x, int y, int box_x, int box_y, int box_w, int box_h) {
++    return ((x >= box_x && x < box_x + box_w) && (y >= box_y && y < box_y + box_h));
```

Comment:
This entire function can be removed now and the checks can go directly into `multimonitor_center_fs()`. :+1:

Reply:
Done.